src/YieldDisburser.sol

```diff
@@ -20,6 +20,14 @@ contract YieldDisburser is OwnableUpgradeable {
 
     event BaseYieldDistributed(uint256 amount, address project);
 
+    error YieldNotResolved();
```

Comment:
alphabetizing stuff like this makes it more readable

Reply:
amended by 1da46aa